### PR TITLE
Update reva to 063b3db9162b

### PR DESCRIPTION
**Pull request by refs**

### CODE REVIEW

**Comment by phil-davis** [comment]
https://drone.owncloud.com/owncloud/ocis/2252/31/6
It got another `Could not resolve host: ocis-server`
This CI timing problem should be resolved by #1101 
In the mean time, I restarted drone CI, it might work when drone agents have all their docker images already cached.

**Review by kulmann — APPROVED** [review]

File: changelog/unreleased/update-reva.md [hunk 1]
```diff
@@ -0,0 +1,7 @@
+Enhancement: Update reva to unsafe tip
```
Comment (micbar) [inline changelog/unreleased/update-reva.md:1]: please change the title

**Comment by phil-davis** [comment]
The problem is https://drone.owncloud.com/owncloud/ocis/2315/68/7
```
Cloning into '/srv/app/web'...
+ cp -r /srv/app/web/tests/acceptance/filesForUpload/* /uploads
+ cd /srv/app/web
+ git checkout a96ce8dfa86dc668b620b609830b75dfa4c07a3d
Note: switching to 'a96ce8dfa86dc668b620b609830b75dfa4c07a3d'.
```

The stupid code copies `filesForUpload` and then checks out a different commit id, which can have different content in `filesForUpload` - so now there are inconsistent `filesForUpload` on the drone agent!!!

**Comment by refs** [comment]
ugh, @phil-davis good catch

**Review by micbar — APPROVED** [review]

**Pull request merged**